addons/l10n_it_edi/__manifest__.py

```diff
@@ -7,9 +7,6 @@
     'version': '0.3',
     'depends': [
         'l10n_it',
-        # Although account_edi is a dependency of account_edi_proxy_client,
-        # it is here because it's in the auto-install
-        'account_edi',
```

Comment:
Did you read the comment?

Reply:
Yes, `account_edi_proxy_client` has already been modified.
https://github.com/odoo/odoo/commit/8590c6a929b49db086b3a1ad64352b7d431e306a